products/appengine/terraform.yaml

```diff
@@ -109,7 +109,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "web_service"
         vars:
           bucket_name: "appengine-test-bucket"
-
+  ServiceSplitTraffic: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{service_id}}"
```

Comment:
Can you try removing the `id_format` and `import_format`? If they need to be there, can you comment with what the format is when they're not set?

We try to make `id` values look like `apps/{{project}}/services/{{service_id}}` now, so that should be the default behaviour. If the generated ones are wrong, you can use that form for both `id` and `import` I think.

Reply:
I tried. But it fails if I try to remove the the id_format and import_format , or change the id_format to apps/{{project}}/services/{{service_id}}. I am doing more testing to find out the reason for that.

Reply:
Can post the error you get when setting `id_format` to `['apps/{{project}}/services/{{service_id}}']`? Using these values is fine if the defaults aren't working.

Reply:
@rileykarson 
It is fixed. Don't get any error with id_format as "apps/{{project}}/services/{{service_id}}".
It was my bad.